### ISSUE TITLE
Error building source

### ISSUE THREAD

**Issue by jmpmscorp** [emqx/emqx#1838]
#### Environment

- OS: Raspbian Desktop x86
- Erlang/OTP: 21 
- EMQ: cloned from https://github.com/emqx/emqx-rel.git

#### Description

Following your make instructions for building emqx this error is showed when make is executed:

```
make[1]: se entra en el directorio '/home/pi/emqx-rel/deps/emqx_coap'
 DEP    gen_coap
error: pathspec 'v0.2.2' did not match any file(s) known to git.
../../erlang.mk:810: fallo en las instrucciones para el objetivo '/home/pi/emqx-rel/deps/gen_coap'
make[1]: *** [/home/pi/emqx-rel/deps/gen_coap] Error 1
make[1]: se sale del directorio '/home/pi/emqx-rel/deps/emqx_coap'
erlang.mk:310: fallo en las instrucciones para el objetivo 'deps'
make: *** [deps] Error 2
```

It seems that gen_coap dependecy is pointing to version 0.2.2 but at github page there isn't gen_coap version 0.2.2, latest is 0.2.1. Could be the reason??

Thank you. Regards.

**Comment by Gilbert-Wong** [comment]
Could you tell me which emqx version do you use?

**Comment by jmpmscorp** [comment]
Hi @Gilbert-Wong. Thanks for you reply.

First of all, I have to apologise. Reading and following rel repository instructions, I copied&pasted git url directly and I didn't know exactly which version and branch I downloaded. When you have replied me, I have gone to search version and I advice that. Consecuently, I have browsed again to rel repository and instead of use git command, I have downloaded 3.0-beta2 version.

Now, I have build source code without errors and I can start broker correctly.

Close thread when you consider.

Thanks again.
